packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts

```diff
@@ -11,14 +11,29 @@ const userSignUp: z.ZodType<UserSignUpRequestDto> = z
 			.min(UserValidationRule.EMAIL_MINIMUM_LENGTH, {
```

Comment:
Could you also call trim (built-in) for all values (name, password, email) and the same for sign in

Reply:
I’ve already called trim() for all the fields, or am I mistaken :thinking:

Reply:
didn't noticed it 😅